style.css

```diff
@@ -0,0 +1,669 @@
+}
+
+#classes-photos-partners-two {
+  margin: 20px;
+  margin-top: 10px;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  margin-left: 20px;
+  margin-bottom: 50px;
+}
+
+.image-partners {
+  margin-right: 45px;
+  margin-bottom: 60px;
+  width: 90px;
+}
+
+#footer {
+  display: flex;
+  flex-direction: row;
+  background-color: rgb(253, 253, 250);
+}
+
+#photo-commons {
+  width: 180px;
+  height: 70px;
+  margin-top: 50px;
+}
+
+#text-footer {
+  color: rgb(245, 77, 48);
+  margin-left: 15px;
+}
+
+#first-text-footer {
+  margin-left: 15px;
+  margin-top: 28px;
+}
+
+.overlay-original {
+  background-color: rgb(240, 98, 73);
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(7px);
+  position: fixed;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  display: flex;
+  display: none;
+}
+
+#menu-main-main {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.menu-cancel {
+  color: #fff;
+  font-size: 29px;
+  font-weight: 500;
+  line-height: 44px;
+  text-decoration: none;
+  padding-top: 50px;
+}
+
+.cancel {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  position: fixed;
+  margin-left: 11px;
+  padding: 30px;
+  margin-top: -8px;
+  color: #fff;
+}
+
+#menu-can1 {
+  position: absolute;
+  height: 24px;
+  left: 150px;
+  right: 68px;
+  font-family: 'Poppins', 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  margin-top: 20px;
+  list-style-type: none;
+}
+
+#menu-can2 {
+  position: absolute;
+  height: 24px;
+  left: 150px;
+  right: 68px;
+  font-family: 'Poppins', 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  margin-top: 110px;
+}
+
+#menu-can3 {
+  position: absolute;
+  height: 24px;
+  left: 140px;
+  right: 68px;
+  font-family: 'Poppins', 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  margin-top: 200px;
+}
+
+#image-contact {
+  width: 50px;
+  margin-left: 10px;
+}
+
+#image-home {
+  width: 30px;
+  margin-left: 10px;
+}
+
+#image-project0 {
+  position: absolute;
+  margin-left: -195px;
+  z-index: -1;
+  margin-top: -18px;
+  width: 60px;
+}
+
+#image-project11 {
+  position: absolute;
+  margin-left: -195px;
+  z-index: -1;
+  margin-top: -18px;
+  width: 60px;
+}
+
+#image-project22 {
+  position: absolute;
+  margin-left: -195px;
+  z-index: -1;
+  margin-top: -18px;
+  width: 60px;
+}
+
+#image-project33 {
+  position: absolute;
+  margin-left: -195px;
+  z-index: -1;
+  margin-top: -18px;
+  width: 60px;
+}
+
+#speakers {
+  margin-left: 45px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
```

Comment:
- [ ] KIndly adjust your margin in order to align the speakers correctly as the provided design.

  - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/191819730-8d7ab947-c6ab-4ba7-ad21-eec3d5b60dd9.png)


  - ### Provided design
![image](https://user-images.githubusercontent.com/87197412/191819834-8bbb4f66-344f-4d7b-bbcc-4452735bc2d8.png)